### DISCUSSION TITLE
Welcome to animated_gif_memory Discussions!

### DISCUSSION THREAD

**Discussion by thelastoutpostworkshop** [thelastoutpostworkshop/animated_gif_memory#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pelso2332** [comment]
Hi - I'm Steve from the UK and have recently started to use the ESP32 to build a few simple projects. I'm happy to share whatever I make, and I'm hoping to learn more about how to exploit this amazing controller!
